src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -194,6 +194,11 @@ void AppHost::_HandleCommandlineArgs()
         // commandline (in the future), it'll trigger this callback, that we'll
         // use to send the actions to the app.
         peasant.ExecuteCommandlineRequested({ this, &AppHost::_DispatchCommandline });
+
+        peasant.DisplayWindowIdRequested({ this, &AppHost::_DisplayWindowId });
+
+        _logic.WindowName(peasant.WindowName());

```

Comment:
how will this work when the window is renamed?

Reply:
A cliffhanger to be resolved on the next episode of "Terminal 1.8: Window Names"? I don't totally remember off the top of my head